src/Cassandra.Tests/MetadataHelpers/SimpleStrategyTests.cs

```diff
@@ -24,9 +24,27 @@ namespace Cassandra.Tests.MetadataHelpers
     public class SimpleStrategyTests
     {
         [Test]
-        public void Should_ReturnThreeReplicasPerToken()
+        public void Should_ReturnTwoReplicasPerToken()
         {
-            var target = new SimpleStrategy(2);
+            var target = new SimpleStrategy(ReplicationFactor.Parse("2"));
+            var testData = ReplicationStrategyTestData.Create();
+
+            var result = target.ComputeTokenToReplicaMap(
+                testData.Ring, testData.PrimaryReplicas, testData.NumberOfHostsWithTokens, testData.Datacenters);
+            
+            // 3 dcs, 3 hosts per rack, 3 racks per dc, 10 tokens per host
+            Assert.AreEqual(10 * 3 * 3 * 3, result.Count);
+
+            foreach (var token in result)
+            {
+                Assert.AreEqual(2, token.Value.Count);
+            }
+        }
+        
+        [Test]
+        public void Should_ReturnTwoReplicasPerToken_When_TransientReplicationIsEnabled()
+        {
+            var target = new SimpleStrategy(ReplicationFactor.Parse("3/1"));
```

Comment:
Replication factor is 3 but expected tokens are 2, is this because of the test data?

Reply:
We basically only consider full replicas when computing token map so in this case there are 2 full replicas and 1 transient replica so the expected replicas count per token is 2

Reply:
oh snap, I was reading it as 3+1, nvm